teaser/project.py

```diff
@@ -80,9 +78,9 @@ def __init__(self, load_data=True):
         self.modelica_info = ModelicaInfo()
 
         self.modelica_project = self.name
-        self.weather_file_header = "wetter"
+        # self.weather_file_header = "wetter"
         self.weather_file_path = "modelica://AixLib/Resources/WeatherData/" \
-                                 "TRY2010_12_Jahr_Modelica-Library.txt"
+                                 "USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos"
```

Comment:
Currently we have no German weather file in the new format available in AixLib or Annex60. As soon as we integrate it, we could switch back to a German file.